design-system/scripts/generate-design-tokens.js

```diff
@@ -290,11 +290,26 @@ fs.writeFileSync(
 // This file is now deprecated and will be remove in the future
 fs.writeFileSync(
   path.join(__dirname, '../materials/custom-properties.css'),
-  prettier.format(printCss(designTokens), {
+  prettier.format(printCss(designTokens.default), {
     ...prettierConfig,
     parser: 'css',
   })
 );
+Object.keys(designTokens).forEach((theme) => {
+  fs.writeFileSync(
+    path.join(__dirname, `../materials/custom-properties_${theme}.css`),
```

Comment:
For the `default` I would keep the file name without the suffix, so just `custom-properties.json`

Reply:
I'm keeping the default `custom-properties.json` as it is right now to be backwards compatible.
I'm only adding two new files with theme suffixes because I think it would be easier for docs team to use them.